app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedListAdapter.kt

```diff
@@ -15,7 +15,7 @@ private const val VIEW_TYPE_SECTION_TITLE_TEXT = 1
 private const val VIEW_TYPE_SECTION_STORY_ITEM = 2
 
 /** Adapter to inflate different items/views inside [RecyclerView] for Ongoing Story List. */
```

Comment:
Update comment to `/** Adapter to inflate different items/views inside [RecyclerView] for Promoted Story List. */`

Reply:
done